stash_engine/app/services/stash_engine/status_dashboard/oai_service.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'httparty'
+
+module StashEngine
+  module StatusDashboard
+
+    class OaiService < DependencyCheckerService
+
+      def ping_dependency
+        super
+        timeframe = (Time.now - 5.minutes).utc.strftime('%y-%m-%dT%H:%M:%SZ')
+        env = Rails.env.production? ? 'prd' : 'stg'
+        base_url = "http://uc3-mrtoai-#{env}.cdlib.org:37001/mrtoai/oai/v2"
```

Comment:
Should this URL be stored in a config file?

Reply:
Probably. Its already in the stash-notifier config (in this repo) but since the dashboard lives in stash_engine it doesn't have access to that config.

Reply:
We should probably do that for all of the hard coded urls/emails to be consistent (e.g. lib/stash/organizations, lib/stash/event_data). 

I'll open another ticket to address that.